docs/PLAN_CASE_RECORD.md

```
@@ -0,0 +1,447 @@
+
+#### レスポンス（409 Conflict）
+```typescript
+{
+  error: "Record has been updated by another session"
+}
+```
+
+### GET /api/case-records/list
+
+#### クエリパラメータ
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)